docs/pages/versions/unversioned/guides/setup-native-firebase.md

```diff
@@ -0,0 +1,125 @@
+  ```
+- Rebuild your iOS project to see the changes: `yarn react-native run-ios`
+
+## Usage with react-native-firebase
+
+After following the iOS and Android setup, you can optionally configure your project to work with `react-native-firebase` version 6 as well.
+
+- Install the `react-native-firebase` packages (e.g. `yarn add @react-native-firebase/app @react-native-firebase/auth etc..`)
+- On iOS, Ensure your `ios/Podfile` has the following lines:
+
+```rb
+  # At the top of the file
+  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+  require_relative '../node_modules/react-native-unimodules/cocoapods'
+
+  # ...
+
+  # Automatically detect installed unimodules
+  use_unimodules!
+
+  # The community version of use_unimodules (used for react-native-firebase)
+  use_native_modules!
+  ```
+- Install the pods on iOS
+
+Continue further on the [react-native-firebase](https://invertase.io/oss/react-native-firebase/) website.
+
+# Managed Workflow Setup
+
+Build your project for the App Store and Play Store and bundle the Google Services info into your application binaries (this does not include support for `react-native-firebase`). Test in development using a custom Expo client on iOS, you cannot test this module locally on Android.
+
+- If you have not already created a Firebase project for your app, do so now by clicking on **Add project** in the [Firebase Console](https://console.firebase.google.com/).
+- In your new project console, create a new native sub-project for iOS and Android.
+
+## Android
+
+- click **Add Firebase to your Android app** and follow the setup steps. **Make sure that the Android package name you enter is the same as the value of `android.package` in your app.json.**
+- Download the config file by clicking **"Download google-services.json"** and drag the file into your Expo project folder.
+- Let the build service know where the file is by defining the relative path in your **`app.json`** with the key `android.googleServicesFile`.
+
+```json
+{
+  "expo": {
+    "android": {
+      "package": "com.mypackage.coolapp",
+      "googleServicesFile": "./google-services.json"
+    }
+  }
+}
+```
+
+- Now create a new build of your project with `expo build:android`
+
+## iOS
+
+- click **Add Firebase to your iOS app** and follow the setup steps. **Make sure that the iOS bundle ID you enter is the same as the value of `ios.bundleIdentifier` in your app.json.**
+- Download the services file by clicking **"Download GoogleService-Info.plist"** and drag this file into your Expo project folder.
+- Let the build service know where the file is by defining the relative path in your **`app.json`** with the key `ios.googleServicesFile`.
+
+```json
+{
+  "expo": {
+    "ios": {
+      "bundleIdentifier": "com.mypackage.coolapp",
+      "googleServicesFile": "./GoogleService-Info.plist"
+    }
+  }
+}
+```
+
+- Now create a new build of your project with `expo build:ios`
```

Comment:
Perhaps also add the option to build a custom IOS client here as well:

- Or create a custom iOS client with your Google Firebase config with `expo client:ios`